src/Device/deviceInfo/deviceInfo.jsx

```diff
@@ -34,10 +34,23 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+import nrf51logoWhite from './nRF51-Series-logo_white.svg';
+import nrf51logoBlue from './nRF51-Series-logo_blue.svg';
+import nrf52logoWhite from './nRF52-Series-logo_white.svg';
+import nrf52logoBlue from './nRF52-Series-logo_blue.svg';
+import nrf53logoWhite from './nRF53-Series-logo_white.svg';
+import nrf53logoBlue from './nRF53-Series-logo_blue.svg';
+import nrf91logoWhite from './nRF91-Series-logo_white.svg';
+import nrf91logoBlue from './nRF91-Series-logo_blue.svg';
+
 const deviceInfo = pcaNumber => ({
```

Comment:
Looking at the device-lister mapping PCA10068 and PCA10115 are missing.

Reply:
I see the entries in https://github.com/NordicSemiconductor/nrf-device-lister-js/blob/facb4aa9e/src/util/board-versions.js but I have no clue what they are. Can you tell me what they are? Or is this something we have not (yet?) released and thus should not (yet) include with details here?

Reply:
These are all the PCA numbers we encountered so far. There are boards with all these numbers, maybe some of them are not available to purchase but they exist.

Reply:
@chunfantasy: I think you added PCA10068 and PCA10115 to the device-lister. Do you have any info on what the boards for those numbers are called and which chip is on them? If not (or if we do not want to disclose it) then I suggest we leave them out. In that case the device selector will show the PCA number instead of the name (before falling back to “Unknown”): https://github.com/NordicSemiconductor/pc-nrfconnect-shared/blob/261b9bd8f81583ce9dbe252e0a3703ca40088a0a/src/Device/DeviceSelector/BasicDeviceInfo.jsx#L47

Reply:
As there is a fallback I'd vote for not holding back the PR for this.